browserslist-stats.json

```diff
@@ -0,0 +1 @@
51988130007,"29":0.007054275596439002,"30":0.0011757125994065003,"31":0.017400546471216204,"32":0.018576259070622707,"33":0.0028217102385756005,"34":0.002586567718694301,"35":0.008465130715726802,"36":0.003527137798219501,"37":0.016224833871809705,"38":0.0016459976391691002,"39":0.016695118911572302,"40":0.007289418116320302,"41":0.005643420477151201,"42":0.015754548832047105,"43":0.027276532306230807,"44":0.005173135437388602,"45":0.015519406312165803,"46":0.031273955144212905,"47":0.03903365830029581,"48":0.07054275596439001,"49":0.4032694215964296,"50":0.20245770961779935,"51":0.030803670104450308,"52":0.05314220949317382,"53":0.4209051105875271,"54":0.19023029858397175,"55":0.25795134430978617,"56":0.48650987363440984,"57":0.39315829324153373,"58":3.4013365500830055,"59":19.189510762473137,"60":21.48215033131581,"61":12.857122702069724,"62":9.220408489585537,"63":3.851399333135814,"64":0.15307778044272635,"65":0.02915767246528121,"66":0},"safari":{"4":0,"5":0.00047028503976260014,"6":0,"7":0.0028217102385756005,"8":0.016930261431453604,"9":0.024925107107417806,"10":0.17588660487121244,"11":1.646467924208863,"3.1":0,"3.2":0,"5.1":0.0007054275596439001,"6.1":0.00023514251988130007,"7.1":0,"9.1":0.13708808909079792,"10.1":3.3084552547298918,"TP":0},"opera":{"15":0,"16":0,"17":0,"18":0,"19":0,"20":0,"21":0,"22":0,"23":0,"24":0,"25":0,"26":0.0009405700795252003,"27":0,"28":0,"29":0,"30":0,"31":0,"32":0.00023514251988130007,"33":0,"34":0,"35":0,"36":0.003527137798219501,"37":0.0018811401590504006,"38":0,"39":0.00047028503976260014,"40":0.00023514251988130007,"41":0,"42":0.0007054275596439001,"43":0.004232565357863401,"44":0.0007054275596439001,"45":0.03926880082017711,"46":0.18435173558693924,"47":0.17917860014955064,"48":0.20034142693886764,"49":0.31650183176022983,"50":0.0023514251988130007,"10.0-10.1":0,"11.5":0,"12.1":0.0032919952783382004},"ios_saf":{"8":0.009640843315133303,"3.2":0,"4.0-4.1":0.00023514251988130007,"4.2-4.3":0.00023514251988130007,"5.0-5.1":0.005878562997032501,"6.0-6.1":0.007054275596439002,"7.0-7.1":0.009640843315133303,"8.1-8.4":0.012227411033827603,"9.0-9.2":0.15119664028367594,"9.3":0.08182959691869242,"10.0-10.2":0.28005474117862833,"10.3":1.3680591806694036,"11.0-11.2":0.5356546602896015},"op_mini":{"all":0.16742147415548564},"android":{"3":0.00023514251988130007,"4":0.005643420477151201,"62":0.15966177099940274,"2.1":0,"2.2":0.00047028503976260014,"2.3":0.003527137798219501,"4.1":0.0009405700795252003,"4.2-4.3":0.0049379929175073015,"4.4":0.023514251988130005,"4.4.3-4.4.4":0},"bb":{"7":0.00047028503976260014,"10":0.0023514251988130007},"op_mob":{"12":0.018576259070622707,"37":0.018576259070622707,"12.1":0.018576259070622707},"and_chr":{"62":2.7224800951856922},"and_ff":{"57":0.07265903864332172},"ie_mob":{"10":0,"11":0},"and_uc":{"11.4":0.40515056175548003},"samsung":{"4":0,"5":0,"6.2":0},"and_qq":{"1.2":0},"baidu":{"7.12":0}},"meta":{"start_date":"2017-06-23","end_date":"2017-12-23"},"uid":"custom.49222282|undefined"}
```

Comment:
Where did you get this data? From our site?

Reply:
@Zmoki from webclient-frontend repo